scss/components/button.scss

```diff
@@ -235,16 +248,16 @@ a.#{$block} {
     @include fd-hover() {
       --fd-button-background-color: var(--fd-color-neutral-1);
       --fd-button-color: var(--fd-color-action-1);
+      --fd-button-border-color: var(--fd-color-action-1);
       @include fd-var-color("color", fd-color("action", 1), --fd-button-color);
       @include fd-var-color("background-color", fd-color("neutral", 1), --fd-button-background-color);
       @include fd-disabled() {
         --fd-button-background-color: transparent;
         --fd-button-color: var(--fd-color-action-1);
         @include fd-var-color("color", fd-color("action", 1), --fd-button-color);
         @include fd-var-color("background-color", transparent, --fd-button-background-color);
-        box-shadow: none;
       }
-      box-shadow: 0 0 0 1px var(--fd-color-action-hover, map-get($fd-colors-action-states, "hover"));
+      @include fd-var-color("border-color", fd-color("action", 1), --fd-button-border-color);
```

Comment:
Why this button used box shadow for hover when default and emphasized didn't, beats me.  🤷‍♂  There are other buttons that do it too, but the entire button styling needs to be overhauled, simplified and brought up to Fiori 3 specs.